src/devices/depthCamera/depthCameraDriver.h

```diff
@@ -97,6 +97,7 @@ namespace yarp
  * |                              |  depthFOV           | double, double      |  Read / write   | degrees        |   -           |  Alternative to HW_DESCRIPTION   | Horizontal and Vertical fields of view of the depth camera                             | 2 values expected as horizontal and vertical FOVs                     |
  * |                              |  rgbMirroring       | bool                |  Read / write   | true/false     |  false        |  Alternative to HW_DESCRIPTION   | Set the mirroring to the acquired rgb image                                            |                                                                       |
  * |                              |  depthMirroring     | bool                |  Read / write   | true/false     |  false        |  Alternative to HW_DESCRIPTION   | Set the mirroring to the acquired rgb image                                            |                                                                       |
```

Comment:
Can you fix also the previous line?
dpeth mirroring is referred to depth image, qhile in the comment is says rgb. I guess it was my copy and paste error